index.html

```diff
@@ -21,7 +21,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ONE</div>
         <div>TWO</div>
         <div>THREE</div>
-        <div>FOUR</div>
+        
+        <div>
+          <img src="https://pbs.twimg.com/profile_images/974056290237534210/dJwwhvu2_400x400.jpg" alt="TK">
+            Tyler Krszjzaniek
+          <ul>
+            <li> <b>email -</b> <a href="mailto:t.krszjzaniek@gmail.com">t.krszjzaniek@gmail.com</a></li>
```

Comment:
`<b>` tags are old-school! We use `<strong>` if we really need semantic markup, but it looks like you _really_ just want to use CSS `bold` for these items.